lib/ajax.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/*
+Plugin Name: Genesis Simple Love
+Plugin URI: https://wordpress.org/plugins/genesis-simple-love
+Description: Add Love feature on Genesis Simple Share Plugin. <strong>This plugin requires Genesis Simple Share. Thanks!</strong>
+Version: 1.0
+Author: phpbits
+Author URI: http://codecanyon.net/user/phpbits/portfolio?ref=phpbits
+License: GPL2
+*/
+if(!class_exists('PHPBITS_GenesisSimpleLove')):
+	class PHPBITS_GenesisSimpleLove{
+		function __construct(){
+			add_action("wp_ajax_genesis_simple_love", array( $this, "ajax_love" ));
+			add_action("wp_ajax_nopriv_genesis_simple_love", array( $this, "ajax_love" ));
+		}
+
+		function ajax_love() {
+			$result = array();
+
+			$nonce = $_REQUEST['nonce'];
+			$post_id = url_to_postid( $_REQUEST['data_url'] );;
+			$loved = array();
+
+	        if ( !wp_verify_nonce( $nonce, "genesis_love")) {
+				exit("You don't have any power here!");
```

Comment:
Make translatable using `genesis-simple-share` namespace, spacing after `(` and before `)`, and preference for `'`